server/controllers/user.controllers.js

```diff
@@ -256,22 +256,24 @@ const UserController = {
       .catch((error) => next(error));
   },
 
-  addDirector(req, res, next) {
+  editDirector(req, res, next) {
     const directorInfo = req.locals;
     const userId = req.user._id;
-    addDirector({ directorInfo, userId })
+    editDirector({ directorInfo, userId })
```

Comment:
I believe it is preferable to send the user object into the edit Director to prevent retrieving the user details again. You might need to update other user variable to `updatedUser`

```suggestion
    const { user } = req;
    editDirector({ directorInfo, user })
```